core/domain/entities/editor/blocks/widgets/EventAttendees.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace EventEspresso\core\domain\entities\editor\blocks\widgets;
+
+use EventEspresso\core\domain\entities\editor\EditorBlock;
+use EventEspresso\core\exceptions\InvalidEntityException;
+
+defined('EVENT_ESPRESSO_VERSION') || exit;
+
+
+
+/**
+ * Class EventAttendees
+ * Returns a list of people that have registered for the specified event
+ *
+ * @package EventEspresso\core\domain\entities\editor\blocks\common
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class EventAttendees extends EditorBlock
+{
+
+    const BLOCK_TYPE = 'widgets-event-attendees';
+
+
+    /**
+     * Perform any early setup required by the block
+     * including setting the block type and supported post types
+     *
+     * @return void
+     */
+    public function initialize()
+    {
+        $this->setEditorBlockType(EventAttendees::BLOCK_TYPE);
+        $this->setSupportedPostTypes(array('espresso_events', 'post', 'page'));
+        $this->setAttributes(array());
+        $this->setAssetRegisterFqcn('EventEspresso\core\domain\entities\editor\blocks\CoreBlocksAssetRegister');
```

Comment:
Might be unneeded if the block is instantiated with an instance of `AssetRegister`?

Reply:
ya that's probably a better way to do this...
I was trying to be clever and do everything via the parent class instead of having to add constructors and declare dependencies for all of the individual blocks, but in light of my big comment above regarding dependencies, this doesn't look so clever does it?